packages/nimble-components/src/rich-text/editor/models/create-tiptap-editor.ts

```diff
@@ -223,6 +223,14 @@ function createCustomMentionExtension(
                 let inSuggestionMode = false;
                 return {
                     onStart: (props): void => {
+                        /**
+                         * If the cursor position moves to the start of the mention and configuration element changes,
+                         * the setMarkdown() will trigger this `onStart` without a decoration node because the cursor
+                         * position is moved out of the suggestion decoration
```

Comment:
```suggestion
                         * If the cursor position moves to outside of the mention and configuration element changes,
                         * the setMarkdown() will trigger this `onStart` without a decoration node because the cursor
                         * position is temporarily moved out of the suggestion decoration. Ignore `onStart` in that case
                         * and don't show the mention list box since it doesn't have anything to anchor to.
```